acq4/devices/AxoPatch200/AxoPatch200.py

```diff
@@ -12,29 +12,16 @@
 from acq4.util.debug import printExc
```

Comment:
Sorry. I am not quite sure how to do it.
`git checkout photon-counting`
`git rebase -i`
??
Thanks.